packages/react-components/react-field/src/components/Field/useFieldStyles.ts

```diff
@@ -75,33 +47,43 @@ const useLabelStyles = makeStyles({
 
   horizontal: {
     marginRight: tokens.spacingHorizontalM,
+    gridRowStart: '1',
+    gridRowEnd: '-1',
     alignSelf: 'start',
     justifySelf: 'stretch',
   },
 });
 
-const useSecondaryTextStyles = makeStyles({
-  base: {
-    marginTop: tokens.spacingVerticalXXS,
-    color: tokens.colorNeutralForeground3,
-    ...typographyStyles.caption1,
-  },
+const useSecondaryTextBaseClassName = makeResetStyles({
+  marginTop: tokens.spacingVerticalXXS,
+  color: tokens.colorNeutralForeground3,
+  ...typographyStyles.caption1,
+});
 
+const useSecondaryTextStyles = makeStyles({
   error: {
     color: tokens.colorPaletteRedForeground1,
   },
-});
 
-const useValidationMessageIconStyles = makeStyles({
-  base: {
-    display: 'block',
-    alignSelf: 'start',
-    fontSize: '12px',
-    lineHeight: '12px',
-    marginRight: tokens.spacingHorizontalXS,
-    marginTop: tokens.spacingVerticalXS,
+  withIcon: {
+    // Add a gutter for the icon, to allow multiple lines of text to line up to the right of the icon.
+    paddingLeft: `calc(${iconSize} + ${tokens.spacingHorizontalXS})`,
   },
+});
+
+const useValidationMessageIconBaseClassName = makeResetStyles({
+  display: 'inline-block',
+  fontSize: iconSize,
+  // Negative left margin puts the icon in the gutter of the validation message div's withIcon style.
+  marginLeft: `calc(-${iconSize} - ${tokens.spacingHorizontalXS})`,
```

Comment:
This seems to work fine, though I haven't tested in Safari. My main concern is that this solution isn't immediately obvious, hence all the comments :)

I know that part of this work is to ensure Field works well with grid auto layout but would flex layout work here? With a little hacking I arrived at the following which I think needs some nudges to get everything perfectly aligned but feels clearer:

```ts
withIcon: {
  display: 'flex',
  // remove paddingLeft
}

// ....
const useValidationMessageIconBaseClassName = makeResetStyles({
  display: 'inline-block',
  fontSize: iconSize,
  marginRight: tokens.spacingHorizontalXS,
  // need a top padding or top margin
});